app/keepers/keepers.go

```diff
@@ -62,6 +63,10 @@ import (
 
 	ibcfeekeeper "github.com/cosmos/ibc-go/v4/modules/apps/29-fee/keeper"
 	ibcfeetypes "github.com/cosmos/ibc-go/v4/modules/apps/29-fee/types"
+
+	ibchooks "github.com/scrtlabs/SecretNetwork/x/ibc-hooks"
+	ibchookskeeper "github.com/scrtlabs/SecretNetwork/x/ibc-hooks/keeper"
```

Comment:
Did we customize the ibc-hooks module for our purposes? why didn't we import it from osmosis but brought all the files here instead?

Reply:
Yeah we needed to change some code that was working with wasmd but not for us (privacy)